video/transcoder/job_test.py

```diff
@@ -132,6 +136,36 @@ def test_create_job_from_preset(capsys, test_bucket):
     assert "Deleted job" in out
 
 
+def test_create_job_from_preset_batch_mode(capsys, test_bucket):
+    create_job_from_preset_batch_mode.create_job_from_preset_batch_mode(
+        project_id, location, input_uri, output_uri_for_preset_batch_mode, preset
+    )
+    out, _ = capsys.readouterr()
+    job_name_prefix = f"projects/{project_number}/locations/{location}/jobs/"
+    assert job_name_prefix in out
```

Comment:
suggestion: please consider using returned value for testing instead of output parsing